department-of-veterans-affairs/caseflow
# Al/appeals 33700 - Update backend to determine no POA versus not listed POA

Resolves [APPEALS-33700](https://jira.devops.va.gov/browse/APPEALS-33700)

# Description
In this ticket, we are updating the backend to nave a `not listed name` of a Power Of Attorney that correlates to the claimant. Also, this information is returned to the front end when viewing the individual decision review screen.

## Acceptance Criteria

## Testing Plan

- rails db:migrate RAILS_ENV=development
- login as a user with Intake authorization with vha business line e.g Acbauervvhah
- Navigate to http://localhost:3000/intake
- selected a High-level review or a supplemental claim 
Continue to search

- input a veteran Id and search
SELECT
- Was this form submitted through VA.gov? yes
- Select a Receipt Date 
- What is the Benefit Type? Veterans Health Administration
- Was an informal conference requested? Yes
- is the claimant someone other than the Veteran? Yes
- Select Claimant not listed
- OPT-IN from SOC/SSOC box on the from? Yes (Soc/SSoc Opt-in)
Continue to the next step

- Select Relationship to the Veteran
- Can be any of the following (Child, Other, Healthcare provider, Attorney)
- Claimant’s or Representative name
- Name not listed 
- Add issue 
- Establish a claim 

- Once successful Go to disposition page  e.g (http://localhost:3000/decision_reviews/vha/tasks/15784)
- under Appellant's Power of Attorney section
- No Known POA.
- inspect section with React Components
- check Task object
- `has_poa: true`
- ```{
  "representative_type": "not_listed",
  ...
}```

https://github.com/department-of-veterans-affairs/caseflow/assets/149511814/f7392c94-7215-4ea0-8f44-6e61013c88ea


# Frontend
## User Facing Changes

 BEFORE|AFTER




 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
